tests/setup.js

```diff
@@ -73,6 +73,11 @@ global.sinon = sinon.createSandbox();
 // Stub the magic constant webpack normally supplies.
 global.CLIENT_CONFIG = require('core/utils').getClientConfig(config);
 
+// See: https://github.com/mozilla/addons-frontend/issues/1138
+global.fetch = () => {
+  throw new Error('API calls MUST be mocked.');
```

Comment:
It would be nice if this message included the URL that was getting fetched since we don't always get a stack trace, especially in sagas. Seeing the URL should help track it down. I guess you'll have to look for `input.url || input` in [fetch()](https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/fetch#Syntax).